src/index.html

```diff
@@ -3,12 +3,79 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li
+            class="nav__item"
+          >
+            <a class="is-active nav__link">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li
+            class="nav__item"
+            data-qa="hover"
+          >
+            <a class="nav__link">
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">
+              Photo
+            </a>
+          </li>
+          <li
+          class="nav__item"
+          id="last"
+          >
```

Comment:
you don't need this id `last` here 
```suggestion
          <li class="nav__item">
```